data-share/data-share-service/src/main/java/io/mosip/datashare/service/impl/DataShareServiceImpl.java

```diff
@@ -119,6 +119,9 @@ public class DataShareServiceImpl implements DataShareService {
 	@Value("${mosip.data.share.protocol}")
 	private String httpProtocol;
 
+	/** Defines whether static data share policy needs to be used for sharing the data*/
+	@Value("${mosip.data.share.static.share.enabled:false}")
+	private boolean staticShareEnabled;

```

Comment:
Lets rename this as follows
mosip.data.share.standalone.mode.enabled
standaloneModeEnabled

Reply:
Lets also create a @postConstruct and add an info log statement that data share is configured in standalone mode, else we can say integrated mode

Reply:
Done

Reply:
@PostConstruct part on logs missing